dipy/tracking/utils.py

```diff
@@ -1014,3 +1019,60 @@ def _as_segments(streamline, break_points):
         yield seg
     for seg in _part_segments(streamline[::-1], break_points[::-1]):
         yield seg
+
+
+def max_angle_from_curvature(min_radius_curvature, step_size):
+    """Get the maximum deviation angle from the minimum radius curvature.
+
+    Parameters
+    ----------
+    min_radius_curvature: float
+        Minimum radius of curvature in mm.
+    step_size: float
+        The tracking step size in mm.
+
+    Return
+    ------
+    max_angle: float
+        The maximum deviation angle in radian,
+        given the radius curvature and the step size.
+
+    References
+    ----------
+    For more information:
+    https://onlinelibrary.wiley.com/doi/full/10.1002/ima.22005
+
+    """
+    max_angle = 2. * np.arcsin(step_size / (2. * min_radius_curvature))
+    if np.isnan(max_angle) or max_angle > np.pi / 2 or max_angle <= 0:
+        max_angle = np.pi / 2.0
+    return max_angle
+
+
+def min_radius_curvature_from_angle(max_angle, step_size):
+    """Get minimum radius of curvature from a deviation angle.
+
+    Parameters
+    ----------
+    max_angle: float
+        The maximum deviation angle in radian.
+        theta should be between [0 - pi/2] otherwise default will be pi/2.
+    step_size: float
+        The tracking step size in mm.
+
+    Return
+    ------
+    min_radius_curvature: float
+        Minimum radius of curvature in mm,
+        given the maximum deviation angle theta and the step size.
+
+    References
+    ----------
+    More information:
+    https://onlinelibrary.wiley.com/doi/full/10.1002/ima.22005
+
+    """
+    if np.isnan(max_angle) or max_angle > np.pi / 2 or max_angle <= 0:
```

Comment:
Same thing here imo. A printed message would be useful.

Reply:
same here, I used a warning